### ISSUE TITLE
Request: Handle starting or ending delimiter

### ISSUE THREAD

**Issue by alphapapa** [magnars/change-inner.el#8]
I don't know why, but for me it seems more natural to choose the closing delimiter than the opening one.  But if I use `change-inner )` instead of `change-inner (`, I get `change-inner*: Couldn't find any expansion starting with ")"`.

Could this be fixed?

**Comment by magnars** [comment]
While possible, it would complicate matters, both the code and conceptually. The simplest fix for you would be to wrap `change-inner` in a function that replaces `)` with `(`, `]` with `[` etc, before calling change-inner itself.

**Comment by alphapapa** [comment]
Oh, I see.  Well, before I (re)discovered your package, I started working on [this](https://gist.github.com/alphapapa/efc7ae427194489f4f07).  I've got it working now so it handles starting or ending delimiters, as well as quotes.  It uses Emacs built-in forward/backward commands, so it avoids having to have some kind of translation table for specific delimiters.

I haven't tested it extensively, but so far it works on Python and Elisp, at least.  :)
